database/schema.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose')
+
+mongoose.connect('mongodb://localhost/lizardboard')
+const db= mongoose.connection
+db.on('error', console.error.bind(console,'connection error:'))
+db.once('open', function() {
+  console.log('connected')
+})
+
+
+const widgetSchema = mongoose.Schema ({
+  type: String, title: String, size: String, contents: String
+})
+const userSchema = mongoose.Schema ({
```

Comment:
Add whitespace. Having said that, two different `Schema`s should probably be defined in separate modules (along with their models, perhaps?).